resources/js/pages/status-pages/components/status-page-form.tsx

```
@@ -130,29 +130,29 @@ export default function StatusPageForm({ statusPage, monitors }: StatusPageFormP
         <div>
           <label className="mb-1 block font-medium text-sm">Logo</label>
```

Biome (2.4.13)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)